library/src/helpers/extractHashData.ts

```diff
@@ -0,0 +1,58 @@
+import { CONTAINER_LABELS } from '../constants';
+
+const CONTAINERS_REGEX = new RegExp(
+  `(.*?)--(${Object.values(CONTAINER_LABELS).join('|')})$`,
+);
+const ITEMS_REGEX = new RegExp(
+  `(.*?)--(${Object.values(CONTAINER_LABELS).join('|')})--(.*?)$`,
+);
+const PROPERTIES_REGEX = new RegExp(`(.*?)--(.*?)$`);
+
+export interface HashData {
+  schema: string;
+  container: CONTAINER_LABELS;
+  item?: string;
+}
+
+function removeHash(str: string): string {
+  return str.startsWith('#') ? str.substr(1) : str;
+}
+
+export function extractHashData(hash: string): HashData | undefined {
+  // hashData[0] = original string
+  // hashData[1] = schema name
+  // hashData[2] = container name
+  let hashData = hash.match(CONTAINERS_REGEX);
+  if (hashData && hashData.length === 3) {
+    return {
+      schema: removeHash(hashData[1]),
+      container: hashData[2] as CONTAINER_LABELS,
+    };
+  }
+
+  // hashData[0] = original string
+  // hashData[1] = schema name
+  // hashData[2] = container name
+  // hashData[3] = rest of string
```

Comment:
those comments are duplicated, remove lines 33,34,35

Reply:
This is not duplicated, because this RegExp is different than above.